weave/trace_server/token_costs.py

```diff
@@ -187,20 +188,28 @@ def get_ranked_prices(
         name=llm_usage_table_alias, cols=[*llm_usage_cols, *derived_llm_usage_cols]
     )
 
-    # Select fields
-    lu_fields = [f"{llm_usage_table_alias}.{col.name}" for col in llm_usage_cols]
     ltp_fields = [
         f"{LLM_TOKEN_PRICES_TABLE.name}.{col.name}" for col in LLM_TOKEN_PRICES_COLUMNS
     ]
+
+    # Clickhouse does not allow parameters in the row_number() over function
+    # This is a temporary workaround, to check the validity of the project_id, to prevent SQL injection
+    validate_project_id(project_id)
```

Comment:
where we validate the project_id

Reply:
I would be even more explicit here: `sql_injection_safe_project_id = ...`